src/main/resources/mybatis/mappers/LanguageMapper.xml

```diff
@@ -17,19 +17,21 @@
     </select>
 
     <insert id="insert" useGeneratedKeys="true" keyColumn="id" keyProperty="id">
-        INSERT INTO languages (name, has_localization)
-        VALUES (#{name}, #{hasLocalization})
+        INSERT INTO languages (name, has_localization, icon)
+        VALUES (#{name}, #{hasLocalization}, #{icon})
```

Comment:
don't forget to take is_default column into consideration on these SQL (insert and update)